pkg/svcat/service-catalog/broker_test.go

```diff
@@ -288,5 +288,13 @@ var _ = Describe("Broker", func() {
 			Expect(actions[1].Matches("update", "clusterservicebrokers")).To(BeTrue())
 			Expect(actions[1].(testing.UpdateActionImpl).Object.(*v1beta1.ClusterServiceBroker).Spec.RelistRequests).Should(BeNumerically(">", 0))
 		})
+		It("Useds the generated v1beta1 Retrieve method to get the broker with namespace", func() {
```

Comment:
Typo is "Useds" (same typo is also a few lines up).

Reply:
ok, will update